### DISCUSSION TITLE
How to stop "AUDIO_INFO"

### DISCUSSION THREAD

**Discussion by mutthunaveen** [schreibfaul1/ESP32-audioI2S#608] [General]
Hello,

wonderful library it is. could be able to play MP3 files in no time. Thanks for most implemented functions which are very much necessary.
Couldnt figureout following. 

how to stop "AUDIO_INFO"? i dont want any debug messages from Library in serial.print; how to avoid them?

could you please support me?

thank you
naveen

**Comment by mutthunaveen** [comment]
any recommendations please?

**Comment by schreibfaul1** [comment]
The functions for the callback events have a "weak" binding. If you don't need them, you can simply leave them out of your program or delete them without the compiler complaining.
